reports/Oregon.md

```diff
@@ -197,6 +197,20 @@ id: or-portland-23
 * https://twitter.com/greg_doucette/status/1268317409095622658
 
 
+### Police shove member of the press | June 4th
+
+In a live broadcast, police are seen shoving a member of the press. She states "An officer Kemmerer stuck his hand to my throat as I held up my press badge." The camera is turned to show that she is clearly labeled press on her ball cap and that she has a badge. The incident occurred at around the 9 minute mark of the 50 minute broadcast. 
+
+tags: press, push, shove
+
+id: or-portland-258
+
+**Links**
+
+* https://twitter.com/i/broadcasts/1ynKOpMqVeqGR
```

Comment:
Little thing but can you timestamp on the broadcast? I know you mention it in the description but just do like a `[9:00](url)` and it'll format it